riak_test/yz_pb.erl

```diff
@@ -244,3 +247,28 @@ confirm_stored_fields(Cluster) ->
     ?assertEqual(Index, proplists:get_value(<<"_yz_rt">>, Fields)),
     ?assertEqual(<<"b1">>, proplists:get_value(<<"_yz_rb">>, Fields)),
     riakc_pb_socket:stop(Pid).
+
+confirm_search_to_test_max_score_defaults(Cluster) ->
+    Index = <<"basic_for_max_score">>,
+    Bucket = {Index, <<"b1">>},
+    create_index(Cluster, Index, Index),
+    lager:info("confirm_search_to_test_max_score_defaults", [Bucket]),
+    Body = <<"{\"age_i\":5}">>,
+    Params = [{sort, <<"age_i asc">>}],
+    HP = select_random(host_entries(rt:connection_info(Cluster))),
+    URL = bucket_url(HP, Bucket, "multivalued"),
+    lager:info("Storing to bucket ~s", [URL]),
+    {Host, Port} = HP,
+    %% populate a value
+    {ok, "204", _, _} = ibrowse:send_req(URL, [?CT_JSON], put, Body),
+    %% Sleep for soft commit
+    timer:sleep(1100),
```

Comment:
I was just keeping to the rest of the set of tests, currently. The initial patch was not due to a race cond.